sympy/series/limits.py

```diff
@@ -136,6 +136,11 @@ def doit(self, **hints):
         if not e.has(z):
             return e
 
+        # limit can work with binomial by converting to factorial first
+        if (e.has(binomial) and not (hints.get('sequence', True)
```

Comment:
yes i tried with name `_eval_rewrite_as_tractable'. And i was grepping for "_rewrite(tractable" as with "_rewrite(factorial" no quotes. that's why i didn't follow that. But i tried with 'e.rewrite('tractable')'. 
And i was unable to reproduce the error with gamma
```
In [9]: e = 16**k / (k * binomial(2*k, k)**2)

In [10]: limit(e.rewrite(gamma), k, oo) # works
Out[10]: π

In [11]: limit(e.rewrite(gamma), k, oo)
Out[11]: π

In [12]: limit(e, k, oo) # This seems okay
Out[12]: π
```